src/print_graph.cc

```diff
@@ -1,13 +1,14 @@
 #include "print_graph.h"
 
 #include <regex>
+#include <sstream>
 
 #include <spdlog/fmt/fmt.h>
+#include <spdlog/fmt/ostr.h>
```

Comment:
Is this include required?

Reply:
Yes, this is required to enable formatting of types that support `operator<<(std::ostream&, T)` . I've added a clarifying comment.